src/pages/components/data-table/usage.mdx

```diff
@@ -429,16 +429,16 @@ The expandable icon always appears first and to the left of the selection icon.
 
 ### Interactions
 
-The data table’s toolbar and rows follow interaction conventions found in other
-Carbon components. Buttons, checkboxes, radio buttons, overflow menus, search,
-and expanding elements are used in the data table, and all behave accordingly to
-their guidelines.
+The data table’s toolbar and rows follow interaction conventions that are found
+in other Carbon components. Buttons, checkboxes, radio buttons, overflow menus,
+search, and expanding elements are used in the data table, and all behave
+according to their guidelines.
 
 #### Mouse
 
 Users can trigger an action or function in the table by clicking the associated
 button or component. Each action or function has its own distinct click target.
-See the accessibility tab for additional interactive guidance.
+See the Accessibility tab for more information.
```

Comment:
Should we hyperlink this? And would the link be just to word "Accessibility" or "Accessibility tab"? 
```suggestion
See the [Accessibility](/components/data-table/accessibility/) tab for more information.
```